interfaces/IBZx.sol

```diff
@@ -406,6 +406,27 @@ interface IBZx {
         uint256 swapAmount, // denominated in collateralToken
         bool returnTokenIsCollateral, // true: withdraws collateralToken, false: withdraws loanToken
         bytes calldata loanDataBytes
+		/* Format:
```

Comment:
there is a natspec format you can follow to document, [here](https://docs.soliditylang.org/en/latest/natspec-format.html)